.github/workflows/cross_browser_e2e_test.yml

```diff
@@ -1,7 +1,6 @@
 name: cross browser e2e test
 permissions:
   contents: read
```

Comment:
Also realized the previous permission were unnecessarily permissive -- I thought the write permissions were needed to update the PR with the status checks, but they aren't (they would be for comments), so removing these throughout